### ISSUE TITLE
Update config roots in README

### ISSUE THREAD

**Issue by heyrict** [oncomouse/coc-bibtex#21]
As I work on another issue in your project, I have found that `workspace.getConfiguration('bibtex')` seems to be located at `coc.preferences.bibtex` rather than `coc.source.bibtex` as you documented in README.md. Probably your documentation should be updated to the latest changes of coc.nvim.

Also, my humble suggestion: As many of the extensions store their config in `coc.preferences.*`,  what do you think about moving `list.source.bibtex.*` to `coc.preferences.bibtex` in accordance with other extensions?

Thank you in advance.

**Comment by oncomouse** [comment]
I don't use coc.nvim anymore, which probably explains why the documentation is out of date. I want to make sure I understand here: everything listed as being stored in `coc.source.bibtex` is now in `coc.preferences.bibtex`? That should be an easy enough fix.

I can fix the other thing, too. The way coc.nvim handled configurations really bugged me when I wrote this, so I'm glad it's more logical.

**Comment by heyrict** [comment]
> everything listed as being stored in coc.source.bibtex is now in coc.preferences.bibtex

Yes, that's it. I have not fully tested other keys (e.g. `list.*`) but they seem unchanged.

Thank you for maintaining this package even after your abandoning `coc.nvim`.

**Comment by oncomouse** [comment]
Sorry for not getting back on this sooner, but I just pushed a branch, `pref-fix`, that implements these changes. Would you mind testing that it's working and I can PR it into the master branch.

EDIT: I just opened a PR, which is referenced below.

**Comment by heyrict** [comment]
Thank you for your update! I'm sorry for that I have provided the wrong information related to the config location change. Most plugins actually stored their configurations [in the object root](https://github.com/neoclide/coc-prettier/blob/master/package.json#L28) rather than in `coc.preferences`. I have pushed a PR (#24) to fix this.

Also, during my testing, changes in `bibtex.triggerCharacters` and `bibtex.triggerPatterns` did not have any effects in my settings. Can you give it a shot?
